multidim-interop/impl/js/v0.46/Makefile

```
@@ -2,7 +2,9 @@ image_name := js-v0.46
 commitSha := b7e608998cc88860d9ec8a3ed7c03fdfb3eccb3b
```

I would suggest updating this to the latest js-libp2p for now

```suggestion
commitSha := 1511105774db3c3c7707809b0262e57bdfa544d8
```

---

Let’s do in a follow up pr. 

This one should just be a fix. I don’t want to debug adding a new version. 

Feel free to create a new pr off this one though